docs/src/main/paradox/reference.md

```diff
@@ -2,7 +2,7 @@
 
 ## API Documentation
 
- * @scaladoc:[Scaladoc](org.apache.pekko.http.scaladsl.index)
+ * @scaladoc:[Scaladoc](org.apache.pekko.http.scaladsl.package-summary)
```

Comment:
We have two api docs paths:

- `https://pekko.apache.org/api/`: has index.html, not package-summary.html
- `https://pekko.apache.org/japi/`: has package-summary.html, not index.html

I think there are some problems with the path under [this page](https://pekko.apache.org/docs/pekko-http/current/reference.html). One links to api and the other is japi. Why?

Reply:
After looking at the Pekko core page, I think it is no problem. Scaladoc links to api (has index page), while javadoc points to japi (no index page)

Reply:
I raised https://github.com/apache/incubator-pekko-http/pull/496